SPRINTLOG.md

```diff
@@ -349,3 +349,7 @@ _Empty sprint_
 # 2024-04-8 - 2024-04-19
 
 - New version: 2.6.4 ([#687](https://github.com/ScilifelabDataCentre/dds_cli/pull/687))
+
+# 2024-05-20 - 2024-06-03
```

Comment:
```suggestion
# 2024-05-20 - 2024-05-31
```
for consistency's sake (we've always had the sprints from starting monday till ending friday)

Reply:
Ah copied the dates from the Jira sprint dates. Fixed now 👍